### ISSUE TITLE
Very great integration

### ISSUE THREAD

**Issue by antoweb** [contemplator1998/smartthings#1]
But can you explain more the documentation especially wath is cvroom?

**Comment by antoweb** [comment]
And wath is camera switch? Seams doesnt work

**Comment by contemplator1998** [comment]
Hey, thanks.
In the latest commits I added all the possible capabilities to be visible (even unsupported ones, which are default for Samsung devices in their API response JSON scheme). 

Feel free to click on any unused/unsupported entity like cvroom / Settings and uncheck Enabled to hide it. 

It's better to have more than less, and then just disabling (instead of stock SmartThings where you don't have actually useful controls/sensors like temperature + 2 doors status + ice maker switch for the refrigerator)

**Comment by antoweb** [comment]
Ok but can you add also'
Reset filter button
Internal camera
Energy Meter

**Comment by contemplator1998** [comment]
I think it's possible with pulling and merging changes from some other HACS SmartThings alternative fork. I saw a Reset filter butter there for my refrigerator. (But I didn't have temperature / ice maker switch / 2 doors status there, so I had to create my own fork). 

Also I don't have a camera to test. Would be great if you could create a pull request with those changes.

**Comment by contemplator1998** [comment]
I think energy sensor is working in my and even stock SmartThings

**Comment by contemplator1998** [comment]
Also you can send any actions right from HA. Check out the official stock SmartThings documentation on how to send  any arbitrary actions.

**Comment by antoweb** [comment]
You can find here code for camera
https://github.com/ibielopolskyi/smartthings_fridge_camera

**Comment by contemplator1998** [comment]
Thanks. Unfortunately I don't have a camera in my refrigerator to test it out. Would be great if you could add it and create a pull request

**Comment by antoweb** [comment]
Unfortunately i'm not a developer and i dont know how to add sensors, entity etc. I would like to use your integration becaudse has two Doors sensors and temperature sensors are working but in the original smartthings integration are presents some Power Energy sensors that now i cant show anymore. 
Is possibile to add those sensors?

**Comment by contemplator1998** [comment]
Right, I see I don't have the power energy on my HA with my fork. I'll try to fix it. Thanks.

**Comment by Roeli1969** [comment]
Hello, thank you very much for your work. What do I need to install for this to work? Do I also need to install "pysmartthings" from dwradcliffe? Do I need to install the normal pysmartthings first?  I have for my heat pump the previously missing component for heating 👍but no longer the one for domestic water. 
Thank you.

**Comment by contemplator1998** [comment]
Hey Roeli1969,
Thanks! Just remove the original integration, add this fork using the button in the README (HACS) and install it. No need to install pysmartthings. This fork doesn't filter for unsupported entities, but rather should add everything as much as possible (leading to some unsupported entities with unknown status, which can be disabled by HA user interface). 

I see there are some discussions going on of reimplementing the original integration, so hopefully it will get better. 

I'll also check the issue of missing power energy sensors in this fork soon. Not sure why they are gone. Anyways, this fork helped me to automate my refrigerator which lacks most of the functionalities in the original integration that I need.

**Comment by Roeli1969** [comment]
Thank you for the clear explanation.
I had done the installation as you suggest.
I now have the other component of the heat pump (heating). I no longer see the domestic water component.
With the original Smartthings integration it is just the other way around.
I can now do what I wanted to do namely turn the heating on and off.
I will keep an eye on your integration. Maybe multicomponent will work for the heat pump in the future.
Greetings.

**Comment by antoweb** [comment]
> Hey Roeli1969, Thanks! Just remove the original integration, add this fork using the button in the README (HACS) and install it. No need to install pysmartthings. This fork doesn't filter for unsupported entities, but rather should add everything as much as possible (leading to some unsupported entities with unknown status, which can be disabled by HA user interface).
> 
> I see there are some discussions going on of reimplementing the original integration, so hopefully it will get better.
> 
> I'll also check the issue of missing power energy sensors in this fork soon. Not sure why they are gone. Anyways, this fork helped me to automate my refrigerator which lacks most of the functionalities in the original integration that I need.

Hello,
Any news about Power consumption sensors?
